benchmarks/benchmarks/bench_lib.py

```diff
@@ -20,6 +20,9 @@ class Pad(Benchmark):
 
     def setup(self, shape, pad_width, mode):
         self.array = np.empty(shape)
+        # explicitly fill the array causing the pagefaults on the provided
+        # array to happen before the actual benchmark starts
+        self.array.fill(2)
```

Comment:
Seems using `np.ones` above would work fine here too?

Reply:
It does. Actually, I just went through the rest of the benchmarks. I used copy() there because I don't know if they actually depend on being zeros.

Is there a place to document this in numpy?